src/lib/structures/game/RoundData.ts

```diff
@@ -0,0 +1,51 @@
+import type { User } from 'discord.js';
+
+export class RoundData {
+	public validTrackNames: string[] = [];
+	public playersGuessedTrackName: Set<User> = new Set();
+	// keys are the valid artists, values are lists of players who have guessed the artist
+	// not the most elegant data structure, I know
+	public trackArtistsGuessed: Record<string, Set<User>> = {};
```

Comment:
Consider making the name less verbose since it's used a lot (`trackArtistsGuessed` -> `artistsGuessed` since it could only ever be for one track)